examples/Notebook/Converting Notebooks With nbconvert.ipynb

```diff
@@ -382,7 +199,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "I'll let you read Jinja manual for the exact syntax of the template."
+    "For details about the template syntax, refer to [Jinja's manual](http://jinja.pocoo.org/docs/dev/)."
```

Comment:
`http://jinja2.readthedocs.org/en/latest/intro.html` ? you link to developement version of the doc.